frontend/src/pages/Causes.jsx

```diff
@@ -3,10 +3,12 @@ import "./styles/Causes.scss";
 import FilterTab from "../components/FilterTab";
 import ProfileCard from "../components/ProfileCard";
 import axios from "axios";
+import { useSelector } from "react-redux";
 const { apiCall } = require("../conf");
 
 function Causes() {
   const [associations, setAssociations] = useState([]);
+  const filter = useSelector(state => state.filter);
```

Comment:
Bad Idea

Reply:
Will fix this in a future PR, and those two branches where respectively made from dev, no worries :) the code is just the same tbh